ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml

```diff
@@ -7,7 +7,17 @@ import "./"
 
 ModalPopup {
     id: popup
-    title: qsTr("New chat")
+
+    header: Text {
+      text: qsTr("New chat")
+      anchors.top: parent.top
+      anchors.left: parent.left
+      font.bold: true
+      font.pixelSize: 17
+      anchors.leftMargin: 16
+      anchors.topMargin: Theme.padding
+      anchors.bottomMargin: Theme.padding
+    }
```

Comment:
It's a bit annoying to have to enter all those anchors when you just want to use a Text like before. maybe we could have a `header` property and also just a `title` property like before. Then you check, if there is a `header`, use that, otherwise, use the `title`

Reply:
Was thinking about that as well. I honestly don't think I fully understood how this content projection works. `header` and `footer` magically work. `content` works completely different (it seems). So I wasn't even sure if we can check for `headers` being available.

Another alternative would be (probably also the route I'd prefer) if we introduce a `SimpleModalPopup` that has a `title` and composes the `Text` element used here implicitly via composition.